Tests/run_tests.py

```diff
@@ -232,12 +232,12 @@ def main(argv):
             import requires_internet
             requires_internet.check.available = False
             # Monkey patch for urlopen()
-            import Bio._py3k
+            import urllib
 
             def dummy_urlopen(url):
                 raise RuntimeError("Internal test suite error, attempting to use internet despite --offline setting")
 
-            Bio._py3k.urlopen = dummy_urlopen
+            urllib.urlopen = dummy_urlopen
```

Comment:
Do we really need this?

Reply:
Yes - it is to make sure ``--offline`` is actually working.